lib/matplotlib/backend_bases.py

```diff
@@ -1006,6 +1006,20 @@ def get_hatch_linewidth(self):
         """Get the hatch linewidth."""
         return self._hatch_linewidth
 
+    def set_hatch_linewidth(self, hatch_linewidth):
+        """Set the hatch linewidth."""
+        self._hatch_linewidth = hatch_linewidth
+
+    def _set_hatch_linewidth(self, hatch_linewidth):
+        if hasattr(self, "set_hatch_linewidth"):
+            self.set_hatch_linewidth(hatch_linewidth)
+        else:
+            _api.warn_deprecated(
```

Comment:
Why is this warning here? 

You define the method that you are `hasattr` checking in the same class as the `_` method, so if the `_` method exists, the public version will exist on `self`. 


Additionally, if there is a warning here, (which I'm not sure a deprecation makes sense here), the removal would be 3.10, not 3.8 (since we already branched for 3.9)

Reply:
I put this in initially thinking there wouldn't be immediate support for all backends. You are right, I think a warning is not necessary. I'll just put in a message like in ```set_alpha``` saying it is not supported in all backends yet. Hatches don't render on wxcairo and pgf.